src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
It is a good tone to set this property for all elements and don't change it without extra need

Reply:
I don't understand what should I do?